### ISSUE TITLE
How to use your classifier for Causal-Norm??

### ISSUE THREAD

**Issue by edwardcho** [hyperconnect/LADE#6]
Hello Sir,

I could solve my issues for LADE and balanced-softmax at your previous response.

But,  I couldn't solve my issues completely.

I can understand your comment for LADE and balanced-softmax.
(Just replacing loss function, I can try it. I can understand)
Thanks...

But, still I didn't solve Causal-Norm in your code.
Should I do create new network using feature-extraction layers and your classifier for other networks ??

Thanks,
Edward Cho.

**Comment by edwardcho** [comment]
Hello Sir,

I saw causal.yaml in config/ImageNet_LT.
Honesty, I don't know how to apply in other network (ex: Inception_v1).

In causal.yaml, I found 'apply_ipca' and 'num_components'.
How to use it other network ??

And, 
for apply to Causal_norm, Should I use 'apply_ipa' & 'num_components' and 'CausalNormClassifier.py' in other network??

Thanks,
Edward Cho.

**Comment by juice500ml** [comment]
Hi! Please check Causal norm's implementation for more details: https://github.com/KaihuaTang/Long-Tailed-Recognition.pytorch
